assets/js/index.js

```diff
@@ -0,0 +1,146 @@
+//~ Declarations
+
+// declaring each working hour in a variable using an object array
+const workingHours = [
```

Comment:
Minor suggestion, you probably don't need the AM/PM on a 24 hour clock in your labels. For instance 14:00 PM should just be 14:00 or 2:00 PM